src/hooks/useSetMobileDevice.ts

```diff
@@ -8,7 +8,7 @@ export const useSetMobileDevice = () => {
 
     React.useEffect(() => {
         const handleResize = () => {
-            const isMobileDeviceCurrent = (window.innerWidth < 768) || (window.innerHeight < 600);
+            const isMobileDeviceCurrent = (window.innerWidth < 768) || (window.innerHeight < 500);
```

Comment:
I suppose to move this value to constants

Reply:
It will be easier to find and change it i future

Reply:
No need to do this, this value will not be changed and used in other place.